mysql_merge/mysql_merger.py

```diff
@@ -336,96 +347,26 @@ def copy_data_to_target(self):
                         row = rows.fetchone()
                         if not row:
                             break
-                        patch.write_line(template.format(row))
+                        for key, value in row.items():
+                            if value is None:
+                                row[key] = 'NULL'
+                        if row[table_map['primary'].keys()[0]] not in ignored_values:
+                            patch.write_line(template.format(row))
                 else:
-                    self._cursor.execute("INSERT IGNORE INTO `%(destination_db)s`.`%(table)s` (%(columns)s) SELECT %(columns)s FROM `%(source_db)s`.`%(table)s` %(where)s" % {
-                        'destination_db': self._destination_db['db'],
-                        'source_db': self._source_db['db'],
-                        'table': table_name,
-                        'where': where,
-                        'columns': "`%s`" % ("`,`".join(columns))
-                    })
+                    self._cursor.execute(
+                        "INSERT IGNORE INTO `%(destination_db)s`.`%(table)s` (%(columns)s) SELECT %(columns)s FROM `%(source_db)s`.`%(table)s` %(where)s" % {
```

Comment:
don't we want to use just insert here as it was proposed in other script?

Reply:
Sure, let me change INSERT for this case